simulation_ws/src/test_nodes/nodes/navigation_test

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+
+import rospy
+import rostest
+import time
+import os
+import unittest
+
+from rosgraph_msgs.msg import Clock
+from ros_monitoring_msgs.msg import MetricList
+from robomaker_simulation_msgs.msg import Tag
+from robomaker_simulation_msgs.srv import Cancel, AddTags
+
+METRIC_NAME = "distance_to_goal"
+
+class SimulationUtils():
```

Comment:
Not sure why you need a class. The state is never used.  Free functions seem sufficient.

Reply:
Fair, removed.

Reply:
I think in a perfect world, we would have a "simulation utils" library that customers could include with their applications, which would likely store some context (and state) about the current simulation job... Interested in your thoughts on this! My original thought for using a class was to create extensible code (so we could easily add more operations in the future) where customers could easily pull the utilities out and use them in their applications.

Perhaps, we should just break out the utilities class as an external python file for now and reference it as an include in the NavigationTest? These two operations (cancel and tag) are not related to the test itself, so makes sense for them to be separate.